server/index.js

```diff
@@ -11,6 +11,7 @@ app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true }));
 app.use(express.static(path.join(__dirname, '/../client/dist')));
 
+//Show a list of alsoBougth
 app.get('/api/alsoBought/:id', (req, res) => {
   Stocks.findRandom().limit(12).exec({ id: req.params.id }, (error, results) => {
     if (error) {
```

Comment:
would recommend looking into express error handling using next, to be more robust